File: Python/CP07.cpp
#include <iostream>
#include <string>
#include <cctype>

int maxConsecutiveDigits(const std::string& str) {
    int currentCount = 0;
    int maxCount = 0;

    for(char ch : str) {
        if(isdigit(ch)) {
            currentCount++;
        } else {
            maxCount = std::max(maxCount, currentCount);
            currentCount = 0;                            
        }
    }

    return std::max(maxCount, currentCount);
}

int main() {
    std::string alfabit = "abc85182defg86fdftimdk391";
    std::cout << "Максимальная длина последовательности цифр: "
              << maxConsecutiveDigits(alfabit) << std::endl;
    return 0;
}

File: Python/CP07.py
def max_consecutive_digits(s):
    current_count = 0 
    max_count = 0  

    for char in s:
        if char.isdigit():
            current_count += 1
        else:
            max_count = max(max_count, current_count)
            current_count = 0  
    
    return max(max_count, current_count)

alfabit = 'abc85182defg86fdftimdk391'
print("Максимальная длина последовательности цифр:", max_consecutive_digits(alfabit))
